app/src/components/Login/Login.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Form, FormGroup, ControlLabel, FormControl, Col, Button }
+from 'react-bootstrap';
+
+let InputGroup = ({id, label, ...props}) => (
+  <FormGroup controlId={id}>
+    <Col componentClass={ControlLabel} sm={12} md={3}>
+      {label}
+    </Col>
+    <Col sm={12} md={9}>
+      <FormControl {...props} required/>
+    </Col>
+  </FormGroup>
+);
+
+export default class Login extends React.Component {
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      email: '',
+      password: ''
+    };
+  }
+
+  handleChange = (stateName) => (e) => {
+    this.setState({[stateName]: e.target.value});
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+  }
+
+  render () {
+    return (
+      <Form horizontal onSubmit={this.handleSubmit}>
```

Comment:
Form horizontal onSubmit={this.handleSubmit}
  {['email', password].map(param => {
      InputGroup id={param} type={param} .......
})}